config/software/agent-deps.rb

```diff
@@ -30,6 +30,11 @@
 # Agent dependencies
 dependency 'boto'
 dependency 'docker-py'
+
+if ENV['JMX_VERSION'] && !ENV['JMX_VERSION'].empty?
```

Comment:
I think we can remove this (for simplicity): when we merge this PR to master, `dd-agent` should be ready for the new jmxfetch build, and if we want to build agents based on bugfix branches we can use the bugfix branches of `dd-agent-omnibus`.

Let me know if you disagree :)